Doc/library/functions.rst

```diff
@@ -1720,50 +1720,86 @@ are always available.  They are listed here in alphabetical order.
    dictionary are ignored.
 
 
-.. function:: zip(*iterables)
-
-   Make an iterator that aggregates elements from each of the iterables.
-
-   Returns an iterator of tuples, where the *i*-th tuple contains
-   the *i*-th element from each of the argument sequences or iterables.  The
-   iterator stops when the shortest input iterable is exhausted. With a single
-   iterable argument, it returns an iterator of 1-tuples.  With no arguments,
-   it returns an empty iterator.  Equivalent to::
-
-        def zip(*iterables):
-            # zip('ABCD', 'xy') --> Ax By
-            sentinel = object()
-            iterators = [iter(it) for it in iterables]
-            while iterators:
-                result = []
-                for it in iterators:
-                    elem = next(it, sentinel)
-                    if elem is sentinel:
-                        return
-                    result.append(elem)
-                yield tuple(result)
-
-   The left-to-right evaluation order of the iterables is guaranteed. This
-   makes possible an idiom for clustering a data series into n-length groups
-   using ``zip(*[iter(s)]*n)``.  This repeats the *same* iterator ``n`` times
-   so that each output tuple has the result of ``n`` calls to the iterator.
-   This has the effect of dividing the input into n-length chunks.
-
-   :func:`zip` should only be used with unequal length inputs when you don't
-   care about trailing, unmatched values from the longer iterables.  If those
-   values are important, use :func:`itertools.zip_longest` instead.
-
-   :func:`zip` in conjunction with the ``*`` operator can be used to unzip a
-   list::
-
-      >>> x = [1, 2, 3]
-      >>> y = [4, 5, 6]
-      >>> zipped = zip(x, y)
-      >>> list(zipped)
-      [(1, 4), (2, 5), (3, 6)]
-      >>> x2, y2 = zip(*zip(x, y))
-      >>> x == list(x2) and y == list(y2)
-      True
+.. function:: zip(*iterables, strict=False)
+
+   Iterate over several iterables in parallel, producing tuples with an item
+   from each one.
+
+   Example::
+
+      >>> for item in zip(['a', 'b', 'c'], [1, 2, 3]):
+      ...     print(item)
+      ...
+      ('a', 1)
+      ('b', 2)
+      ('c', 3)
+
+   More formally: :func:`zip` returns an iterator of tuples, where the *i*-th
```

Comment:
You use <code>:func:\`zip\`</code> like 10 times throughout this section (basically every time you refer to `zip`).

And all it does is link back to... this same section (my browser doesn't even move, so it feels like a broken link). I would consider just using <code>\`\`code()\`\`</code> formatting for all of them. For example:

```suggestion
   More formally: ``zip()`` returns an iterator of tuples, where the *i*-th
```

Reply:
I'm okay with making this change, but I worry that it wouldn't be consistent with the other sections of the documentation. @gvanrossum What do you think?

Reply:
Leave it as is. This is a common phenomenon.